constraint/pkg/client/drivers/local/local.go

```diff
@@ -492,3 +497,192 @@ func (d *driver) Dump(ctx context.Context) (string, error) {
 
 	return string(b), nil
 }
+
+func MapModules(templ *templates.ConstraintTemplate, extern []string) (string, []string, error) {
```

Comment:
Add a comment which describes the intent of "MapModules". It is not clear what the intent is in general, or what "externs" mean for the function.

Reply:
Per comment in client.go, this can be made package-private.

Reply:
Moved back to `driver.AddTemplate`, we might need a validation function to resolve module parsing/validation in CreateCRD in the immediate future.